actors.md

```diff
@@ -477,6 +478,16 @@ func CommitSector(sectorID SectorID, commD, commR, commRStar []byte, proof SealP
 		miner.ProvingPeriodEnd = chain.Now() + ProvingPeriodDuration(miner.SectorSize)
 	}
 }
+
+func collateralForPower(power BytesAmount) TokenAmount {
+  availableFil = FakeGlobalMethods.GetAvailableFil()
+  totalNetworkPower = StorageMinerActor.GetTotalStorage()
+  numMiners = FakeGlobalMethods.GetTotalNumMiners()
+  powerCollateral = availableFil * NetworkConstants.POWER_COLLATERAL_PROPORTION * power / totalNetworkPower
+  perCapitaCollateral = availableFil * NetworkConstants.PER_CAPITA_COLLATERAL_PROPORTION / numMiners
```

Comment:
We should specify truncation towards zero as the rounding mode (assuming that intent).

Reply:
I think it makes sense to round up. If the "true" collateral required is (say) 1.3, then by holding 2 FIL you are over-collateralized (fine) but by holding 1 you are under collateralized (technically disallowed).

We can round after summing the two collateral types.

Reply:
This arithmetic is all integer arithmetic of attofil. The `/` operator means truncation by default almost everywhere. Even if we specified decimal arithmetic, we would still need to define a specific precision and rounding mode. It's ok to specify rounding up, but the `ceil` below doesn't do anything to integers.

Rounding other than truncation towards zero will generally be harder to implement. For example, Go's `big.Int.Div` only truncates (but we can simulate the other modes with `DivMod`).